scripts/tests/doppelganger_protection.sh

```diff
@@ -41,28 +35,31 @@ exit_if_fails cp -R $HOME/.lighthouse/local-testnet/node_1 $HOME/.lighthouse/loc
 echo "Starting bootnode"
 
 exit_if_fails ../local_testnet/bootnode.sh &> /dev/null &
-BOOT_PID=$!
+
+exit_if_fails ../local_testnet/el_bootnode.sh &> /dev/null &
 
 # wait for the bootnode to start
 sleep 10
 
+echo "Starting local execution nodes"
+
+exit_if_fails ../local_testnet/geth.sh $HOME/.lighthouse/local-testnet/geth_datadir1 7000 6000 5000 $genesis_file &> geth.log &
+exit_if_fails ../local_testnet/geth.sh $HOME/.lighthouse/local-testnet/geth_datadir2 7100 6100 5100 $genesis_file &> /dev/null &
+exit_if_fails ../local_testnet/geth.sh $HOME/.lighthouse/local-testnet/geth_datadir3 7200 6200 5200 $genesis_file &> /dev/null &
+
+sleep 20
+
 echo "Starting local beacon nodes"
 
-exit_if_fails ../local_testnet/beacon_node.sh $HOME/.lighthouse/local-testnet/node_1 9000 8000 &> /dev/null &
-BEACON_PID=$!
-exit_if_fails ../local_testnet/beacon_node.sh $HOME/.lighthouse/local-testnet/node_2 9100 8100 &> /dev/null &
-BEACON_PID2=$!
-exit_if_fails ../local_testnet/beacon_node.sh $HOME/.lighthouse/local-testnet/node_3 9200 8200 &> /dev/null &
-BEACON_PID3=$!
+exit_if_fails ../local_testnet/beacon_node.sh -t $PEER_ID -d debug $HOME/.lighthouse/local-testnet/node_1 9000 8000 http://localhost:5000 $HOME/.lighthouse/local-testnet/geth_datadir1/geth/jwtsecret &> beacon1.log &
```

Comment:
I don't see reference of `PEER_ID` and the `-t` flag anywhere, is something missing here?

Reply:
ahh nice catch, it was remnants of some previous ugliness.